modules/yarn.lock

```diff
@@ -10603,11 +10603,6 @@ interpret@^2.0.0:
   resolved "https://registry.yarnpkg.com/interpret/-/interpret-2.0.0.tgz#b783ffac0b8371503e9ab39561df223286aa5433"
   integrity sha512-e0/LknJ8wpMMhTiWcjivB+ESwIuvHnBSlBbmP/pSb8CQJldoj1p2qv7xGZ/+BtbTziYRFSz8OsvdbiX45LtYQA==
 
-intersection-observer@0.11.0:
```

Comment:
Interesting... do you thing this is related to the updated, or a leftover?

Reply:
My assumption is that it's a leftover, but I'll check after I've finished my current `ant all`.